corgi/tasks/management/commands/loaderratadata.py

```diff
@@ -73,26 +73,28 @@ def handle(self, *args, **options) -> None:
                 f"api/v1/erratum/search?show_state_SHIPPED_LIVE=1&product[]={product_id}",
                 page_data_attr="data",
             )
-            all_errata = set()
-            for erratum in product_errata:
-                self.stdout.write(self.style.SUCCESS(f"Found shipped erratum {erratum['id']}"))
-                builds = et.get(f"api/v1/erratum/{erratum['id']}/builds_list.json")
-                for product_version in builds.values():
-                    for build in product_version["builds"]:
-                        for _, build_data in build.items():
-                            for variant in build_data["variant_arch"].keys():
-                                if variant in options["product_variants"]:
-                                    self.stdout.write(
-                                        self.style.SUCCESS(
-                                            f"Found variant {variant} in {erratum['id']}"
-                                        )
-                                    )
-                                    all_errata.add(erratum["id"])
-
-            for erratum in all_errata:
-                self.stdout.write(self.style.SUCCESS(f"Loading erratum: {erratum}"))
-                slow_load_errata.delay(erratum)
+            self.load_errata_with_variants(et, options["product_variants"], product_errata)
 
         else:
             self.stderr.write(self.style.ERROR("No errata IDs or repo flag, or variants specified"))
             sys.exit(1)
+
+    def load_errata_with_variants(self, et, product_variants, product_errata):
+        for erratum in product_errata:
+            has_variants = False
+            self.stdout.write(self.style.SUCCESS(f"Found shipped erratum {erratum['id']}"))
+            builds = et.get(f"api/v1/erratum/{erratum['id']}/builds_list.json")
+            for product_version in builds.values():
+                for build in product_version["builds"]:
+                    for _, build_data in build.items():
```

Comment:
If we don't need the key value here, let's use build.values() instead of build.items():
```suggestion
                    for build_data in build.values():
```

Reply:
This was existing code, leaving it for later.